docs/diagrams/AddModuleActivityDiagram.puml

```diff
@@ -0,0 +1,31 @@
+@startuml
+skin rose
+skinparam ActivityFontSize 15
+skinparam ArrowFontSize 12
+!pragma useVerticalIf on
+start
+:User executes add command;
+
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+:Parse commandString;
+
+if () then ([else])
+    :ParseException: Invalid Command Format;
+    stop
```

Comment:
Can try adding puml partitions (AddCommandParser and AddCommand) to show where everything happens.